dockers/docker-bgp/minigraph_facts.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python
```

Comment:
There are some concerns about
1. duplicate the minigraph parsers in every containers.
2. runtime overhead to generate config files on each bootup

The requirement is as:
1. there is no change of minigraph.xml after deployment
2. so, it is possible to create config files on host, copy to docker containers after they are just created. only the first bootup will be slower (if possible, we can run this step before reboot)